### PR TITLE
15 shield hit12a expand with dose to water and let to water scorers

### LINKED ISSUE #13: upload particle weighted spotlists, converted from dicom plans
Work ongoing here:

https://github.com/DataMedSci/pymchelper/pull/563

### PR DESCRIPTION
**Pull request by nbassler**
fixes #13 and #15

### CODE REVIEW

File: data/sh12a/generate_results.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+# function convert{
+#         convertmc
+# }
+# set -o xtrace
+
+
+# basenames for 1-d plots
+bplot="NB_Z_narrow_dose NB_Z_narrow_dose_water NB_Z_narrow_LET NB_Z_narrow_LET_water NB_Z_narrow_QEFF"
```
Comment (grzanka) [inline data/sh12a/generate_results.sh:10]: Why using `NB` everywhere ?

Comment (nbassler) [reply data/sh12a/generate_results.sh:10]: it is very convenient for fast scripting to have a prefix for the resulting files, irrespective of suffix.

And I like the letters NB :)

Comment (grzanka) [reply data/sh12a/generate_results.sh:10]: OK, thanks

**Review by grzanka — APPROVED** [review]
LGTM

**Pull request merged**